back/app/controllers/web_api/v1/custom_pages_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+class WebApi::V1::CustomPagesController < ApplicationController
+  skip_before_action :authenticate_user, only: %i[index show by_slug]
```

Comment:
Shouldn't we authenticate the user also when updating & destroying actions are performed?